### DISCUSSION TITLE
Cartographer on a Delta printer

### DISCUSSION THREAD

**Discussion by titschai** [community/community#116317] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello everyone, sorry for my clumsy approach. This is my first time doing something like this and I only have a rudimentary command of English
I am thrilled with the cartographer. I prepared it for installation in my Delta printer (FLSUN v400).
Now I'm looking for a way to integrate it into software.
I already have it installed in Klipper (I think), but how do I design the printer.cfg.
The problem is that home-z isn't the printing plate and I can't use sensorless homing here.
For bed mesh leveling, you always have to attach an additional sensor to this printer, which also uses a different pin than the z-endstop.
Here is an excerpt from my printer.cfg:

########################################
# Z Stepper Motor & Driver Settings
########################################

[stepper_c]
step_pin: PB5
dir_pin: PB4
enable_pin: !PB8
microsteps: 64  # If you encounter an error while measuring the resonances. Replace the microsteps to 16 just for the duration of the test.
rotation_distance: 40
endstop_pin: ^PC4
angle: 90

[tmc2209 stepper_c]
uart_pin: PD4
run_current: 1.2
interpolate: False
stealthchop_threshold: 0

########################################
# Probe Settings
########################################
#serial: /dev/serial/by-id/usb-Cartographer_614e_310023000A43304253373820-if00 
[probe]
pin: !PA11
x_offset: 0
y_offset: 0
z_offset: 0
speed: 10
lift_speed: 50
samples: 3
samples_result: average
sample_retract_dist: 6
samples_tolerance: 0.02
samples_tolerance_retries: 5

########################################
# Delta Calibration & Mesh Settings
########################################

[delta_calibrate]
radius: 147
horizontal_move_z: 30
speed: 100

[bed_mesh]
speed: 100
horizontal_move_z: 30
mesh_radius: 147
mesh_origin: 0,0
mesh_pps: 4,4
round_probe_count: 9
algorithm: bicubic
bicubic_tension: 0.1
move_check_distance: 3
zero_reference_position: 0,0
#fade_start: 1
#fade_end: 30 

It would be nice if you could help me with this (and maybe some others with Delta printers). :-)
Best regards
Thomas

**Comment by Akash1134** [comment]
Hi @titschai ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
